web/src/components/tasks/CheckStatus.tsx

```diff
@@ -48,7 +48,17 @@ export const CheckStatus = (props: Props): ReactElement => {
   useMountEffectWhenDefined(() => {
     if (!business) return;
 
-    if (business.licenseData) {
+    if (business.formationData.formationResponse?.success) {
```

Comment:
[boulder] Please correct me if I'm misunderstanding the logic flow, but based on how these `if` statements are structured, it appears the component will always render with the `formationData`. So, if a user were to, for instance, edit the pre-filled information, check their status, and then click `edit` again, wouldn't their newly inputted information be overwritten by the data from formation? Shouldn't we first check whether they have existing information in the `licenseData` before auto-filling from `formationData`? If I recall correctly from our discussion during the planning session, we agreed that `licenseData` should be the source of truth for this feature.

Reply:
Thanks for the feedback/reminder!
Updated